semanticdb/metap/src/main/scala/scala/meta/cli/Metap.scala

```diff
@@ -218,4 +445,40 @@ object Metap {
     Ordering.by(_.range)
   private implicit def synthOrder: Ordering[Synthetic] =
     Ordering.by(_.range)
+
+  private def rep[T](pre: String, xs: Seq[T], sep: String, suf: String)(f: T => Unit): Unit = {
```

Comment:
@ShaneDelmore `rep`/`opt` were almost directly stolen from your AST prettyprinters in Rsc. Without them, writing the SemanticDB prettyprinter would be much less pleasant. 👍